src/components/Card/Card.test.tsx

```diff
@@ -0,0 +1,238 @@
+                    <div>Written in English</div>
+                    <div>Under Creative Commons License</div>
+                </>
+            }
+        ></Card>
+    );
+
+    let cardWithNoContent = (
+        <Card
+            id="card#1"
+            extraClassName={["edition-card"]}
+            image={
+                <Image
+                    src="https://placeimg.com/300/400/arch"
+                    isDecorative={true}
+                />
+            }
+            heading={
+                <Heading
+                    level={2}
+                    id="editioncardheading1"
+                    url="#edition-link"
+                    text={"2004 Edition"}
+                />
+            }
+            ctas={
+                <div className="edition-card__ctas">
+                    <Link linkType={LinkTypes.Button} href="blah">
+                        Read Online
+                    </Link>
+                    <div className="edition-card__download">
+                        <Link href="#url" linkType={LinkTypes.Action}>
+                            <Icon
+                                name="download"
+                                blockName="more-link"
+                                decorative={true}
+                                modifiers={["left"]}
+                                iconRotation={IconRotationTypes.rotate0}
+                            ></Icon>
+                            Download
+                        </Link>
+                    </div>
+                </div>
+            }
+        ></Card>
+    );
+
+    let cardWithNoImage = (
+        <Card
+            id="card#1"
+            extraClassName={["edition-card"]}
+            heading={
+                <Heading
+                    level={2}
+                    id="editioncardheading1"
+                    url="#edition-link"
+                    text={"2004 Edition"}
+                />
+            }
+            content={<div>Required middle column content</div>}
+            ctas={
+                <div className="edition-card__ctas">
+                    <Link linkType={LinkTypes.Button} href="blah">
+                        Read Online
+                    </Link>
+                    <div className="edition-card__download">
+                        <Link href="#url" linkType={LinkTypes.Action}>
+                            <Icon
+                                name="download"
+                                blockName="more-link"
+                                decorative={true}
+                                modifiers={["left"]}
+                                iconRotation={IconRotationTypes.rotate0}
+                            ></Icon>
+                            Download
+                        </Link>
+                    </div>
+                </div>
+            }
+        ></Card>
+    );
+
+    it("Generates a Card with an optional header, footer, image, middle content, and CTAs", () => {
```

Comment:
I'd drop optional in the test description here since it's not really testing the "optionality".